.github/workflows/agent-build.yml

```diff
@@ -20,11 +20,13 @@ on:
       - main
       - 'php8'
       - 'dev'
+      - 'clm-obd'
```

Comment:
We likely want to remove the changes for this file now that clm-obd is merging into dev

Reply:
agreed
```suggestion
```